src/main/java/fi/dy/masa/minihud/config/InfoToggle.java

```diff
@@ -15,60 +15,77 @@
+    TIME_WORLD              ("infoTimeWorld",               false, "", "minihud.config.info_toggle.comment.infoTimeWorld", "minihud.config.info_toggle.name.infoTimeWorld"),
+    TIME_WORLD_FORMATTED    ("infoWorldTimeFormatted",      false, "", "minihud.config.info_toggle.comment.infoWorldTimeFormatted", "minihud.config.info_toggle.name.infoWorldTimeFormatted"),
+
+    // Player (Camera)
+    COORDINATES             ("infoCoordinates",             true,  "", "minihud.config.info_toggle.comment.infoCoordinates", "minihud.config.info_toggle.name.infoCoordinates"),
+    COORDINATES_SCALED      ("infoCoordinatesScaled",       false, "", "minihud.config.info_toggle.comment.infoCoordinatesScaled", "minihud.config.info_toggle.name.infoCoordinatesScaled"),
+    BLOCK_POS               ("infoBlockPosition",           false, "", "minihud.config.info_toggle.comment.infoBlockPosition", "minihud.config.info_toggle.name.infoBlockPosition"),
+    CHUNK_POS               ("infoChunkPosition",           false, "", "minihud.config.info_toggle.comment.infoChunkPosition", "minihud.config.info_toggle.name.infoChunkPosition"),
+    BLOCK_IN_CHUNK          ("infoBlockInChunk",            false, "", "minihud.config.info_toggle.comment.infoBlockInChunk", "minihud.config.info_toggle.name.infoBlockInChunk"),
+    DIMENSION               ("infoDimensionId",             false, "", "minihud.config.info_toggle.comment.infoDimensionId", "minihud.config.info_toggle.name.infoDimensionId"),
+    FACING                  ("infoFacing",                  true,  "", "minihud.config.info_toggle.comment.infoFacing", "minihud.config.info_toggle.name.infoFacing"),
+    ROTATION_YAW            ("infoRotationYaw",             false, "", "minihud.config.info_toggle.comment.infoRotationYaw", "minihud.config.info_toggle.name.infoRotationYaw"),
+    ROTATION_PITCH          ("infoRotationPitch",           false, "", "minihud.config.info_toggle.comment.infoRotationPitch", "minihud.config.info_toggle.name.infoRotationPitch"),
+
+    // Player
+    BLOCK_BREAK_SPEED       ("infoBlockBreakSpeed",         false, "", "minihud.config.info_toggle.comment.infoBlockBreakSpeed", "minihud.config.info_toggle.name.infoBlockBreakSpeed"),
+    SPEED                   ("infoSpeed",                   false, "", "minihud.config.info_toggle.comment.infoSpeed", "minihud.config.info_toggle.name.infoSpeed"),
+    SPEED_AXIS              ("infoSpeedAxis",               false, "", "minihud.config.info_toggle.comment.infoSpeedAxis", "minihud.config.info_toggle.name.infoSpeedAxis"),
+    SPEED_HV                ("infoSpeedHV",                 false, "", "minihud.config.info_toggle.comment.infoSpeedHV", "minihud.config.info_toggle.name.infoSpeedHV"),
+    SPRINTING               ("infoSprinting",               false, "", "minihud.config.info_toggle.comment.infoSprinting", "minihud.config.info_toggle.name.infoSprinting"),
```

Comment:
The Sprinting should be last, because it would look kind of Janky with it sandwiched in the middle, IMO.